neuralmonkey/server/server.py

```diff
@@ -15,7 +16,44 @@
 APP.config["args"] = None
 
 
+def root_dir():  # pragma: no cover
+    return os.path.abspath(os.path.dirname(__file__))
+
+
+def get_file(filename):  # pragma: no cover
+    src = os.path.join(root_dir(), filename)
+    return open(src).read()
+
+
+def run(data):  # pragma: no cover
+    args = APP.config["args"]
+    dataset = Dataset("request", data, {})
+    # TODO check the dataset
+    # check_dataset_and_coders(dataset, args.encoders)
+
+    _, response_data = run_on_dataset(
+        args.tf_manager, args.runners,
+        dataset, args.postprocess, write_out=False)
+
+    return response_data
+
+
 @APP.route("/", methods=["GET", "POST"])
+def index():
+    if request.method == "POST":
+        source_text = request.form["source"]
+        data = {"source": [source_text.split()]}
+        translation_response = run(data)
+        translation = " ".join(translation_response["target"][0])
+    else:
+        source_text = "enter tokenized soruce language text here ."
+        translation = ""
+
+    return render_template(
+        "server.html", translation=translation, source=source_text)
+
+
+@APP.route("/translate", methods=["POST"])
```

Comment:
Tomuhle bych taky říkal run.

Reply:
tzn nezapomenout to změnit tady a ještě v tests/tests_run.sh

Reply:
mate to tam